server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java

```diff
@@ -73,6 +73,7 @@ public String toString() {
     GRANT_USER_PRIVILEGE, REVOKE_USER_PRIVILEGE, GRANT_USER_ROLE, REVOKE_USER_ROLE, CREATE_ROLE,
     DELETE_ROLE, GRANT_ROLE_PRIVILEGE, REVOKE_ROLE_PRIVILEGE, LIST_USER, LIST_ROLE,
     LIST_USER_PRIVILEGE, LIST_ROLE_PRIVILEGE, LIST_USER_ROLES, LIST_ROLE_USERS,
-    GRANT_WATERMARK_EMBEDDING, REVOKE_WATERMARK_EMBEDDING, DELETE_STORAGE_GROUP,
+    GRANT_WATERMARK_EMBEDDING, REVOKE_WATERMARK_EMBEDDING,
+    TTL, DELETE_STORAGE_GROUP
```

Comment:
Why put  TTL before DELETE_STORAGE_GROUP instead of adding it to the end.

Reply:
See the previous reply.